jdk-1.7-parent/annotation/src/main/java/org/wicketstuff/annotation/scan/AnnotatedMountScanner.java

```diff
@@ -97,46 +98,16 @@
 {
 
 	/**
-	 * Get the Spring search pattern given a package name or part of a package name
-	 * 
-	 * @param packageName
-	 *            a package name
-	 * @return a Spring search pattern for the given package
-	 */
-	public String getPatternForPackage(String packageName)
```

Comment:
Maybe it would be better to mark it as deprecated and leave the old impl. Just to make migration to 7.x easier. We will drop all deprecated methods with 8.c